Content.Server/Weapons/Melee/Shield/EnergyShieldComponent.cs

```diff
@@ -0,0 +1,33 @@
+using Robust.Shared.Audio;
+
+namespace Content.Server.Weapons.Melee.EnergyShield;
+
+[RegisterComponent]
+internal sealed class EnergyShieldComponent : Component
```

Comment:
Is there a reason this needs to be in Server? If all of the events exist in shared this should be moved to Shared.

Reply:
When we get predicted hitscan this will need to be in shared.

Reply:
If you translate to General, it falls off 
`using Content.Server.CombatMode.Disarm;`
`using Content.Server.Kitchen.Components;`
 , they support only the Server.

Reply:
Why do you need Kitchen for Energy Shield? Either way you can still put the component in shared while the system is in server.

Reply:
done